clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -1298,6 +1298,18 @@ def GlobalOp : CIR_Op<"global", [Symbol]> {
 
       return isDeclaration();
     }
+
+    /// Whether the definition of this global may be replaced at link time.  NB:
+    /// Using this method outside of the code generators is almost always a
+    /// mistake: when working at the IR level use isInterposable instead as it
+    /// knows about ODR semantics.
+    bool isWeakForLinker()  {
```

Comment:
This is already available in `clang/include/clang/CIR/Dialect/IR/CIROpsEnums.h`

Reply:
@bcardosolopes I left the method as a wrapper to the `isWeakForLinker` in `CIROpsEnums`. I didn't want to fully remove it so that we can still make linkage query directly to the operations.